### PR TITLE
draft

### PR DESCRIPTION
**Pull request by yesC48**
not complete :)

### CODE REVIEW

**Comment by yesC48** [comment]
please merge this

File: assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+
+
 mod.ol.contributors = [#7f7f7f]Сontributors:[] \n[#735384]Dj Silin[], [#e4b400]MissionaryPoseBest[], [#3498db]SUS[], yesC48, [#008A00]Uuj[#FFFF00]uju[].
 mod.ol.description = [#badbeb]My first mod. Adds mostly unique content.[]
 mod.ol.musics = [#7f7f7f]Music:[] \nComing soon!
 mod.ol.subtitle1 = [#87ceeb]he he
-mod.ol.subtitle2 = [#87ceeb]I`m alive!
-mod.ol.subtitle3 = [#87ceeb]:З
+mod.ol.subtitle2 = [#87ceeb]I'm alive!
+mod.ol.subtitle3 = [#87ceeb]yesC48 was here
```
Comment (xStaBUx) [inline assets/bundles/bundle.properties:15]: ...
Leave it as it was

File: assets/bundles/bundle.properties [hunk 2]
```diff
@@ -23,70 +25,57 @@ setting.mod.ol.show.description = Removes warning window that appears after star
...
 item.ol-grumon.description = Used in defense and production.
 item.ol-grumon.details = A metal with strong magnetic abilities.
-
 item.ol-omalite.name = Omalite
 item.ol-omalite.description = Used in production, ammunition and advanced coolant.
 item.ol-omalite.details = Cold, radioactive stone. Source unknown.
-
 item.ol-omalite-alloy.name = Omalite Alloy
-item.ol-omalite-alloy.description = Widely used in buildings.
+item.ol-omalite-alloy.description = Widely used in buildings within the mod.
```
Comment (xStaBUx) [inline assets/bundles/bundle.properties:40]: It is undesirable for the mod to be mentioned in the item description

File: assets/bundles/bundle.properties [hunk 3]
```diff
@@ -23,70 +25,57 @@ setting.mod.ol.show.description = Removes warning window that appears after star
...
-block.ol-omalite-alloy-wall.description = Protects buildings from enemy shells. Freezes units that have laser weapons if they hit a wall. Resilient. Doesn't burn.
-
+block.ol-omalite-alloy-wall.description = Protects buildings from enemy shells.\nFreezes units that have laser weapons if they hit a wall. Resilient. Doesn't burn.
 block.ol-omalite-alloy-wall-large.name = Large Omalite Alloy Wall
-block.ol-omalite-alloy-wall-large.description = Protects buildings from enemy projectiles. Freezes units that posses a laser weapon, provided they hit the wall. Doesn't burn.
-
+block.ol-omalite-alloy-wall-large.description = Protects buildings from enemy projectiles.\nFreezes units that posses a laser weapon, provided they hit the wall. Doesn't burn.
 block.ol-test-joint.name = Test join block
-
+block.ol-test-joint.description = Protects structures from enemy projectiles. Visually merges with blocks of the same type.\nSandbox only.
```
Comment (xStaBUx) [inline assets/bundles/bundle.properties:68]: This block will be removed in upcoming updates.
This is just a test

File: assets/bundles/bundle.properties [hunk 4]
```diff
@@ -97,7 +86,7 @@ stat.magnetic = Magnetism
 stat.average-surface-temperature = Average surface temperature
 unit.kelvins = [#bfbfbf]K[]
 stat.radius = Radius
-unit.kilometers = [#bfbfbf]mill. km[]
+unit.kilometers = [#bfbfbf]km[]
```
Comment (xStaBUx) [inline assets/bundles/bundle.properties:89]: km³

File: assets/bundles/bundle.properties [hunk 5]
```diff
@@ -63,8 +63,8 @@ block.ol-deep-dalanii.name = Deep Dalanii
 block.ol-omalite-alloy-wall.name = Omalite Alloy Wall
 block.ol-omalite-alloy-wall.description = Protects buildings from enemy shells.\nFreezes units that have laser weapons if they hit a wall. Resilient. Doesn't burn.
 block.ol-omalite-alloy-wall-large.name = Large Omalite Alloy Wall
-block.ol-omalite-alloy-wall-large.description = Protects buildings from enemy projectiles.\nFreezes units that posses a laser weapon, provided they hit the wall. Doesn't burn.
-block.ol-test-joint.name = Test join block
+#block.ol-omalite-alloy-wall-large.description = Protects buildings from enemy projectiles.\nFreezes units that posses a laser weapon, provided they hit the wall. Doesn't burn.
```
Comment (xStaBUx) [inline assets/bundles/bundle.properties:66]: what did you comment on the description of the large wall?

**Pull request closed without merging**